doc/reference/reference_sql/sql_statements_and_clauses.rst

```diff
@@ -620,140 +620,90 @@ causes no error message, although (s1, s1) is probably a user error.
-    CREATE TABLE f2 (
-        ordinal INTEGER PRIMARY KEY,
-        referring_planet STRING DEFAULT 'Earth',
-        FOREIGN KEY (referring_planet) REFERENCES f1 (referenced_planet)
-            ON UPDATE SET DEFAULT
-            ON DELETE CASCADE);
-    INSERT INTO f1 VALUES (1, 'Mercury'), (2,' Venus'), (3, 'Earth');
-    INSERT INTO f2 VALUES (1, 'Mercury'), (2, 'Mercury');
-    UPDATE f1 SET referenced_planet = 'Mars'
-        WHERE referenced_planet = 'Mercury';
-    SELECT * FROM f2;
-    DELETE FROM f1 WHERE referenced_planet = 'Earth';
-    SELECT * FROM f2;
-    ... In this example, the UPDATE statement changes the referenced key,
-        and the clause is ON UPDATE SET DEFAULT, therefore both of the
-        rows in f2 have referring_planet set to their default value,
-        which is 'Earth'. The DELETE statement deletes the row that
-        has 'Earth', and the clause is ON DELETE CASCADE,
-        therefore both of the rows in f2 are deleted.
+    {referencing_column_name} {column_definition}
+        REFERENCES {referenced_table_name}({referenced_column_name})
 
-Limitations: |br|
-* Foreign keys can have a MATCH clause (`Issue#3455 <https://github.com/tarantool/tarantool/issues/3455>`_).
+.. NOTE::
 
-.. COMMENT
-   Constraint Conflict Clauses are temporarily disabled.
-   However, the description is here, as a big comment.
+    With :doc:`2.11.0 </release/2.11.0>`, the following referencing options aren't supported anymore:
 
-   Constraint Conflict Clauses
+    *   The ``ON UPDATE`` and ``ON DELETE`` triggers. The ``RESTRICT`` trigger action is used implicitly.
+    *   The ``MATCH`` subclause. ``MATCH FULL`` is used implicitly.
+    *   ``DEFERRABLE`` constraints. The ``INITIALLY IMMEDIATE`` constraint check time rule is used implicitly.
 
-   In a CREATE TABLE statement:
-   CREATE TABLE ... constraint-definition ON CONFLICT {ABORT | FAIL | IGNORE | REPLACE | ROLLBACK} ...;
+Note that a referenced column should meet one of the following requirements:
 
-   In an INSERT or UPDATE statement:
-   {INSERT|UPDATE} OR {ABORT | FAIL | IGNORE | REPLACE | ROLLBACK} ...;
+-   A referenced column is a PRIMARY KEY column.
+-   A referenced column has a UNIQUE constraint.
+-   A referenced column has a UNIQUE index.
 
-   The standard way to handle a constraint violation is "statement rollback" -- all rows affected by the statement are restored to their original values -- and an error is returned. However, Tarantool allows the user to specify non-standard ways to handle PRIMARY KEY, UNIQUE, CHECK, and NOT NULL constraint violations.
+**Example**
 
-   ABORT -- do statement rollback and return an error. This is the default and is recommended, so a user's best strategy is to never use constraint conflict clauses.
+This example shows how to create a relation between the parent and child tables through a single-column foreign key:
 
-   FAIL -- return an error but do not do statement rollback.
+1.  First, create a parent ``author`` table:
```

Comment:
Nit: "first", "then", and similar words are usually redundant in procedures with numbered steps.
Let's keep it shorter